modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/snapshot/storage/SnapshotStorage.scala

```diff
@@ -181,6 +181,10 @@ object SnapshotStorage {
             case None    => snapshotLocalFileSystemStorage.read(hash)
           }
 
+        def getHash(ordinal: SnapshotOrdinal): F[Option[Hash]] = get(ordinal).flatMap(
+          _.map(_.toHashed.map(_.hash)).traverse(identity)
```

Comment:
as above

Reply:
but also I think it might be just `_.traverse(_.toHashed.map(_.hash))` but I may be wrong

Reply:
Changed

```scala
_.map(_.toHashed.map(_.hash)).traverse(identity)
```

to

```scala
_.traverse(_.toHashed.map(_.hash))
```